schemas/basicDataTypes/array/containsValidation.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "array",
+  "contains": {
+    "type": "number"
+  }

```

Comment:
Specified in draft 6, `contains` keyword is defined as

> while the items schema must be valid for every item in the array, the contains schema only needs to validate against one or more items in the array. 

I included a test case of this since it is specified in the JSON Schema docs,
but I am not sure how this should be displayed in our schema viewer at the moment.
Should this also be a case we should consider testing or drop completely?

Reply:
Found that Taskcluster actually uses the `contains` keywords in their schemas, so should definitely test this as well!

`metaschema.json` uses
```json
"contains": {
     "$ref": "#/definitions/schema"
},
```

Reply:
Good find. Yes, it's a good idea to have this tested.